src/sass/header.sass

```diff
@@ -0,0 +1,16 @@
+.header
+  display: flex
+  justify-content: space-between
+  padding: 32px 80px
+
+  img
+    margin-top: 8px
+
+  .links
+    display: flex
+    gap: 25px
```

Comment:
Nice

Reply:
Thank you!!

Reply:
Turns out gap isn't widely supported in combination with flex: https://caniuse.com/?search=gap

Reply:
Huh, that's interesting. I wonder if it's in the works. Surprised Safari and Opera don't.